cfn-resources/stream-instance/test/contract-testing/cfn-test-create-inputs.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Is this file needed? What is the difference with ./test/cfn-test-create-inputs.sh

Reply:
In other resource this is done. We are keeping consistency between different resources, also it makes it very simple to include tests in CI and it's possible to run all tests and input generations from the make file